src/fmu/dataio/scripts/initialize_case.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+
+"""
+
+Initialize an FMU case with metadata and Sumo registration (optional).
```

Comment:
I think "initialize case" is a somewhat misleading term in this context. The term "case" is already an ERT context, and I would initially believe that ERT is the one responsible for initializing cases.

Reply:
Could call it something like `create_case_metadata`, perhaps. @jcrivenaes

Reply:
I suggest we consistently rename from "initialize case" to "create case metadata" throughout the code, starting with the `InitializeCase` class name. Suggest to do this in separate PR to compartmentalize it.

Reply:
Renamed as far as it gets in this PR and this script. Suggest a separate PR for renaming further into `fmu-dataio`.

Reply:
#171